### PR TITLE
Update superpmi.py script for new 'runtime' repo layout

### PR DESCRIPTION
**Pull request by BruceForstall**
Updated a few comments as well.

### CODE REVIEW

File: src/coreclr/scripts/superpmi.py [hunk 1]
```diff
@@ -2100,7 +2117,7 @@ def main(args):
     # await/async requires python >= 3.5
```
Comment (jashook) [inline src/coreclr/scripts/superpmi.py]: Do you mind changing the comment to >= 3.8

File: src/coreclr/scripts/superpmi.py [hunk 2]
```diff
@@ -2100,7 +2117,7 @@ def main(args):
...
     if sys.version_info.major < 3 and sys.version_info.minor < 5:
```
Comment (jashook) [inline src/coreclr/scripts/superpmi.py]: Also could you change the check to sys.version_info.minor < 8

**Review by jashook — APPROVED** [review]
Other than that lgtm thank you!

File: src/coreclr/scripts/superpmi.py (hunk 1 above)
Comment (BruceForstall) [reply src/coreclr/scripts/superpmi.py]: Do you want to force 3.8+? I made the comment say 3.8, but I'm not sure it actually requires anything more than 3.5 (you said it needs that for async)

**Comment by BruceForstall** [comment]
I went ahead and merged. If you want to force 3.8+ instead of 3.5+ as it is now, I'll do that in a separate change.

File: src/coreclr/scripts/superpmi.py (hunk 1 above)
Comment (jashook) [reply src/coreclr/scripts/superpmi.py]: It technically requires 3.7 as 3.6 on windows has an async bug.

**Comment by jashook** [comment]
We should force 3.7

**Pull request merged**